CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 Change Log
 ==========
 
+0.11.0
+------
+
+* Add install_certificate command, enabling consumers to download certificates
+  for environments from a remote certificate store.
```

Comment:
This isn't part of the library, I don't think it needs a new version (or mention in CHANGELOG)